src/collector/Backend.cpp

```diff
@@ -54,8 +53,6 @@ void Backend::clone_from(const Backend & other)
 
     std::memcpy(&m_stat, &other.m_stat, sizeof(m_stat));
     std::memcpy(&m_calculated, &other.m_calculated, sizeof(m_calculated));
```

Comment:
Please, add a comment to both methods which will describe why they do what they do.